apps/web/src/app/components/CollateralTypeSwitcher/SingleChainItem.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { Token } from '@balancednetwork/sdk-core';
+import { Typography } from 'app/theme';
+import { Position, XChainId } from 'app/pages/trade/bridge/types';
+import { useTheme } from 'styled-components';
+import { xChainMap } from 'app/pages/trade/bridge/_config/xChains';
+import { AssetSymbol, BalanceAndValueWrap, DataText } from '../Wallet/styledComponents';
+import { StyledListItem } from './MultiChainItem';
+import { toFraction } from 'utils';
+import { useOraclePrices } from 'store/oracle/hooks';
+import { t } from '@lingui/macro';
+import CurrencyLogoWithNetwork from '../CurrencyLogoWithNetwork';
+import { formatValue } from 'utils/formatter';
+
+type SingleChainItemProps = {
+  baseToken: Token;
+  networkPosition: Partial<{ [key in XChainId]: Position }>;
+  onSelect: (symbol, chainId) => void;
```

Comment:
add types

Reply:
moved to https://github.com/balancednetwork/balanced-network-interface/issues/1388